build.sbt

```diff
@@ -149,6 +148,7 @@ lazy val reflect = projectMatrix
     moduleName := "scalafix-reflect",
     isFullCrossVersion,
     libraryDependencies ++= Seq(
+      semanticdbScalacCore,
```

Comment:
I wonder how much of the current implementation of `GlobalSymbolTable` works against class files produced by dotty.... We haven't got any issue but I guess it's just because the Symbol lookup API is not used much on  symbols that are not in the document.

https://github.com/scalacenter/scalafix/blob/c6d741174abf76c01862a2aac1b52379b106c5d9/scalafix-core/src/main/scala/scalafix/internal/v1/InternalSemanticDoc.scala#L80

Reply:
https://github.com/scalacenter/scalafix/issues/2049